src/main/java/org/folio/circulation/domain/Request.java

```diff
@@ -28,13 +28,15 @@
 
 import java.util.Objects;
 
+import lombok.EqualsAndHashCode;
 import org.apache.commons.lang3.StringUtils;
 import org.joda.time.DateTime;
 
 import io.vertx.core.json.JsonObject;
 import lombok.AllArgsConstructor;
 
 @AllArgsConstructor
+@EqualsAndHashCode
```

Comment:
Why does this class need to override equals and hash code?

Reply:
This is an artifact. Can be safely deleted.

Reply:
Ok, please remove it and resolve this comment when it has been removed.